src/ArachneTest.cc

```diff
@@ -62,7 +60,7 @@ struct Environment : public ::testing::Environment {
 
         CoreArbiterServer::sys = sys;
         coreArbiterServer = new CoreArbiterServer(
-                std::string("/tmp/CoreArbiter/testsocket"),
+                std::string(TEST_SOCKET),
                 std::string("/tmp/CoreArbiter/testmem"),
```

Comment:
Since you're converting static strings to macros or constants, you may want to go ahead and change this one too.